engine/common/net_encode.c

```diff
@@ -885,63 +885,63 @@ int Delta_ClampIntegerField( int iValue, qboolean bSigned, int bits )
 		iValue = bound( 0, (byte)iValue, 1 );
 		break;
 	case 2:
-		if( bSigned ) iValue = bound( -1, (short)iValue, 2 );
+		if( bSigned ) iValue = bound( -2, (short)iValue, 1 );
 		else iValue = bound( 0, (word)iValue, 3 );

```

Comment:
It should be like that (or maybe I missed something), because 1 bit for sign, and x bit for value, so for 2bits, 1 bit value is 0-1 for positive, and -1 -2 for negative.